config/jobs/periodic/docker-in-docker/periodic-dind-build.yaml

```diff
@@ -0,0 +1,213 @@
+          PATH_GITHUB="https://github.com/florencepascual/${DIR_GITHUB}.git"
+          
+          # path to the image for building and testing
+          PATH_IMAGE_BUILD="quay.io/florencepascual/docker_ce_build"
+
+          # path to the scripts 
+          PATH_SCRIPTS="/workspace/${DIR_GITHUB}"
+          DATE=`date +%d%m%y-%H%S`
+          PATH_LOG_PROWJOB="/workspace/prow-job-${DATE}.log"
+
+          export DIR_GITHUB
+          export PATH_SCRIPTS
+          export PATH_LOG_PROWJOB
+
+          # Create the log file
+          echo "Create the prow job log file"
+          touch ${PATH_LOG_PROWJOB}
+          if ! test -f ${PATH_LOG_PROWJOB}
+          then
+            echo "The log file ${PATH_LOG_PROWJOB} was not created."
+            exit 1
+          fi
+
+          # clone the directory where the scripts are
+          echo "* Git clone and get the dockerd-entrypoint script*" 2>&1 | tee -a ${PATH_LOG_PROWJOB}
+          git clone ${PATH_GITHUB}
+          if ! test -d ${DIR_GITHUB}
+          then
+            echo "The directory from ${PATH_GITHUB} was not cloned." 2>&1 | tee -a ${PATH_LOG_PROWJOB}
+            exit 1
+          fi
+          wget -O ${PATH_SCRIPTS}/dockerd-entrypoint.sh https://raw.githubusercontent.com/docker-library/docker/master/dockerd-entrypoint.sh 
+          if ! test -f ${PATH_SCRIPTS}/dockerd-entrypoint.sh
+          then
+            echo "The dockerd-entrypoint file was not downloaded." 2>&1 | tee -a ${PATH_LOG_PROWJOB}
+            exit 1
+          fi
+          chmod a+x ${PATH_SCRIPTS}/*.sh
+
+          # start the dockerd
+          echo "** Dockerd **" 2>&1 | tee -a ${PATH_LOG_PROWJOB}
+          # bash ${PATH_SCRIPTS}/dockerd-entrypoint.sh &
+          source ${PATH_SCRIPTS}/dockerd-starting.sh
+
+          if [ -z "$pid" ]
+          then
+            echo "There is no docker daemon." 2>&1 | tee -a ${PATH_LOG_PROWJOB}
+            exit 1
+          else
+            # get the env file and the dockertest repo and the latest built of containerd if we don't want to build containerd
+            echo "*** COS Bucket ***" 2>&1 | tee -a ${PATH_LOG_PROWJOB}
+            CONT_NAME=docker_s3_env
+            docker run --env S3_SECRET_AUTH --env PATH_LOG_PROWJOB -d -v /workspace:/workspace --privileged --name $CONT_NAME debian:bullseye /bin/bash -c "${PATH_SCRIPTS}/get_env.sh"
+            status_code="$(docker container wait $CONT_NAME)"
+            echo $status_code 2>&1 | tee -a ${PATH_LOG_PROWJOB}
+
+            if [[ ${status_code} -ne 0 ]]
+            then
+              echo "The docker to get the env.list and the dockertest has failed." 2>&1 | tee -a ${PATH_LOG_PROWJOB}
+              docker logs $CONT_NAME 2>&1 | tee -a ${PATH_LOG_PROWJOB}
+              exit 1
+            fi
+
+            ls /workspace 2>&1 | tee -a ${PATH_LOG_PROWJOB}
+            
+            if [[ -f env.list ]]
+            then
+              # check there are 3 env variables in env.list
```

Comment:
Why not checking of 3 env variables in `env.list` should be placed in `get_env.sh` shell script itself.

Reply:
True, I will change that